CHANGES.txt

```diff
@@ -1,5 +1,70 @@
 ﻿NOTE: A combined description of changes since NUnit 2.6.3 is under development along with a summary statement of major features. Both will be available in the first Beta release.
 
+NUnit 3.0.0 Beta 1 - In Process
+
+General
+
+ * There is now a master windows installer for the framework, engine and console runner.
+
+Framework
+
+ * We no longer create a separate framework build for .NET 3.5. The 2.0 and
+   3.5 builds were essentially the same, so the former should now be used 
+   under both runtimes.
+ * A new Constraint, DictionaryContainsKeyConstraint, may be used to test
+   that a specified key is present in a dictionary.
+ * LevelOfParallelizationAttribute has been renamed to LevelOfParallelism.
```

Comment:
Attribute should be removed from the first or added to the second for consistency.